springboard-retail.gemspec

```diff
@@ -1,8 +1,8 @@
 Gem::Specification.new do |s|
   s.name              = 'springboard-retail'
-  s.version           = '4.1.1'
+  s.version           = '4.2.0'
```

Comment:
You don't need to bump the version or otherwise modify the gemspec in this PR.

Reply:
Roger roger